document/static/js/modules/exporter/print/index.js

```diff
@@ -35,6 +35,8 @@ export class PrintExporter extends HTMLExporter {
             }
             section:footnote-content {
                 display: block;
+                font-style:normal;
+                font-weight:normal;
             }
```

Comment:
could you also add text-decoration? I assume this is also true for underline

Reply:
but do we have underline option in fidus?

Reply:
Yes. If you enable it in the document template designer.

Reply:
Okay, sure I will make the necessary changes